bc_ccv_rstudio/form.yml

```diff
@@ -11,8 +11,14 @@ form:
   - r_module
   - bc_email_on_started
 attributes:
-
+  bc_account:
+    label: "Condo account"
+    help: "You can leave this blank if you are not using a condo else enter the condo name here for example cbc-condo"
   bc_num_slots: 1
+  bc_queue:
+    value: ""
+    label: "Partition"
+    help: "You can leave this blank if you are **not** using the default, else select the partition here"
```

Comment:
I'm confused here. If I am _not_ using the default partition, I leave this blank? 

Wouldn't I leave it blank _only if_ I'm using the default partition?

Reply:
If you are on a condo will it default to that condo?

Reply:
if your default queue is a condo then it will default to that. Else you can specify the condo with the `condo account`